interface/weno/pharmacyupdates.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
where is this script used?

Reply:
This process has not been complete. This is just the groundwork to get there. They are still working out the details.

Reply:
i wouldn't include code/script that isn't completed (guessing when this feature is ready will need to do that crazy password/encryption scheme anyways, so will need a new/updated script for this anyways in a future patch)

Reply:
@juggernautsei , still need to address this